pkg/api/admin/openshiftcluster.go

```diff
@@ -130,14 +130,24 @@ const (
 	MTU3900 MTUSize = 3900
 )
 
+// OutboundType represents the type of routing a cluster is using
+type OutboundType string
+
+// OutboundType constants
+const (
+	OutboundTypeUserDefinedRouting  OutboundType = "UserDefinedRouting"
+	OutboundTypeLoadbalancerRouting OutboundType = "LoadbalancerRouting"
```

Comment:
I think we just want to call this `Loadbalancer` not `LoadbalancerRouting`.  

Maintains consistency with https://github.com/openshift/installer/blob/master/pkg/types/azure/platform.go#L18